README.md

```diff
@@ -66,18 +49,111 @@ For users familiar with Windmill, Flow offers additional advanced features that
+`postgres://DB_USER:DB_PASS@localhost:5432/DB_NAME`
+
+### Manual Deployment
+
+> **Note:**
+> 
+> This method can also be used for enterprise or custom scaled Windmill setups.
+
+**Prerequisites:**
+
+- Nextcloud instance with `AppAPI` and `webhooks_listener` apps enabled.
+- Python 3.10 or higher installed on the server.
+- Git installed on the server.
+
+Follow these steps to manually deploy Flow without Docker:
+
+1. **Register the Deploy Daemon in AppAPI**
+
+   First, register the Deploy Daemon in AppAPI with the `manual-install` type. Refer to the Nextcloud documentation for detailed instructions: [AppAPI Manual Install](todo)
+
+2. **Clone the Repository and Install Dependencies**
+
+   Clone the Flow repository:
+
+   ```bash
+   git clone https://github.com/cloud-py-api/flow.git
+   ```
+
+   Navigate into the cloned directory, create a Python virtual environment, and install the required dependencies:
+
+   ```bash
+   cd flow
+   python3 -m venv venv
+   source venv/bin/activate
+   pip install -r requirements.txt
+   ```
+
+3. **Install Windmill**
+
+   Install Windmill by following the official instructions: [Setup Windmill on Localhost](https://www.windmill.dev/docs/advanced/self_host#setup-windmill-on-localhost)
+
+   **Note:** Windmill should be in its default state, with the default user `admin@windmill.dev` and password `changeme`. If you have changed the administrator username/password, adjust the `DEFAULT_USER_EMAIL` and `DEFAULT_USER_PASSWORD` variables in the `ex_app/lib/main.py` file accordingly.
+
+4. **Configure ExApp Environment Variables**
+
+   Set the required ExApp environment variables (`NEXTCLOUD_URL`, `APP_HOST`, `APP_PORT`, `APP_SECRET`) or modify them directly in the script at the top of `ex_app/lib/main.py`. Refer to the Nextcloud documentation for more details: [ExApp Configuration](todo)
+
+5. **Set the `WINDMILL_URL` Environment Variable**
+
+   Define the `WINDMILL_URL` environment variable or set it in the script. This variable specifies the location of the Windmill instance you deployed in step 3.
+
+6. **Adjust Windmill Version (Optional)**
+
+   **Note:** If you are using a non-standard version of Windmill from the ExApp, please adjust its version in the `Makefile` by editing the line under the `init` target:
+
+   ```makefile
+   git -c advice.detachedHead=False clone -b v1.394.4 https://github.com/windmill-labs/windmill.git windmill_src
+   ```
+
+   Replace `v1.394.4` with your desired Windmill version.
+
+7. **Initialize Windmill Source and Build Frontend**
+
+   Run the following commands from the cloned `flow` repository to clone Windmill into the `windmill_src` folder and build the frontend:
+
+   ```bash
+   make init
+   make static_frontend
+   ```
+
+   These commands will:
+
+   - Clone the Windmill source code into `windmill_src`.
+   - Build the frontend assets and create the `static_frontend` folder, which will be served by ExApp as the Windmill frontend.
```

Comment:
Does this work with the enterprise version of Windmill? I thought the sources were not available.

Reply:
> Does this work with the enterprise version of Windmill? I thought the sources were not available.

As far as I suspect, the frontend sources should be the same?
If not, then when your order a subscription for Enterprise Windmill customer need to ask Windmill org to also build and provided the frontend with two flags, like we do in Dockerfile